oliviaruizknott/timer-rails
# Changes display of past items’ elapsed time

Stripes!

![screen shot 2018-07-04 at 12 56 06 pm](https://user-images.githubusercontent.com/26205183/42288664-a8207498-7f89-11e8-9a38-a2b5a4c272fa.png)

## To Test
1. Start a meeting, say, this one: https://salvadortimer.herokuapp.com/meetings/80
2. Let the timer run long enough on the first item that you can see a good amount of the elapsed time (it will be orange while the item is current).
3. When you click "Next Item", you’ll see stripes where there used to be orange! There should also be some space around the text.
